index.html

```diff
@@ -2316,11 +2316,39 @@ <h2>Privacy Considerations</h2>
       <section>
         <h3>Unlinkability</h3>
 
-        <p class="issue">
-When the contents of a digitally signed payload contains correlatable
-identifiers, those identifiers can be used to track individuals. A static
-digital signature is a correlatable identifier. There are digital signature
-schemes that provide uncorrelatable digital signatures.
+        <p>
+When the contents of a digitally signed payload contains a correlatable
+identifier, that identifier can be used for tracking purposes by a verifier,
+which might lead to a violation of an expected level of privacy. An example of
+such an identifier is a shopping loyalty card number. However, some of these
+identifiers might not be immediately apparent, such as a static digital
+signature or a cryptographic hash of an image, which can also be used as a
+correlatable identifier.
```

Comment:
Some wordsmithing to:

* Bring in a definition for "correlatable" to replace perhaps redundant statement
* Better signal that some correlation might actually be expected (and not unwanton)
* Decouple a loyalty card number from the unwanton correlation case since people might want their purchasing behavior to be correlated in some fashion for better loyalty rewards

```suggestion
When a digitally-signed payload contains an identifier that is seen by multiple
verifiers, it becomes a point of correlation. An example of such an identifier is
a shopping loyalty card number. Correlatable identifiers can be used for tracking
purposes by verifiers, which can sometimes violate privacy expectations. The
fact that some identifiers can be used for tracking might not be immediately
apparent. Examples of such correlatable identifiers include, but are not limited
to, a static digital signature or a cryptographic hash of an image.
```

Reply:
what about expanding past "identifier" to be "a piece or pieces of data that may be seen by multiple verifiers" -- foreseeably even if an `id` property is not shared, there could be n other fields that give sufficient information in order to correlate.

Reply:
@dlongley — I *think* you meant [`and not unwanton` and `unwanton correlation`](https://github.com/w3c/vc-data-integrity/pull/52/files#r973764166) to say `and not unwanted` and `unwanted correlation`, respectively?

Reply:
Further on @dlongley's suggestion, in line 2321, I'd change `it becomes a point` to `that identifier becomes a point` (because the `it` *could* refer back to the entire payload, which I think is not intended).

Reply:
+1 to replace `identifier` with a more general term. "a piece or pieces of data", tried rewording Dave's statement
```suggestion
When a digitally-signed payload contains a piece or pieces of data that are seen by multiple
verifiers, it becomes a point of correlation. An example is
a shopping loyalty card number. Correlatable data can be used for tracking
purposes by verifiers, which can sometimes violate privacy expectations. The
fact that some pieces of data can be used for tracking might not be immediately
apparent. Examples of such correlatable data include, but are not limited
to, a static digital signature or a cryptographic hash of an image
```